spec/unit/inputs/kafka_spec.rb

```diff
@@ -83,6 +83,18 @@
     it "should register" do
       expect { subject.register }.to_not raise_error
     end
+
+    context "x" do
```

Comment:
This test looks unfinished - can you fix this up before committing

Reply:
Fixed with 644898366ea6eff051d094f67d0a20060448b0ec